account/tests.py

```diff
@@ -338,3 +338,127 @@ def test_password_is_only_number(self):
 
         self.assertEqual(f.is_valid(), False)
         self.assertEqual(f.errors['password2'][0], "このパスワードは数字しか使われていません。")
+
+
+class LoginTest(TestCase):
+    """
+    ログイン機能に対するテスト
+    """
+
+    def setUp(self):
+        account = Account.objects.create_user(
+            email='sample@example.com', 
+            username='sample'
+        )
+        account.set_password('instance1')
+        account.save()
+        self.path = reverse('account:login')
+        self.client = Client()
+
+    def test_correct_login(self):
+        """
+        正しくログインした場合
+        """
+
+        saved_accounts = Account.objects.all()
+        self.assertEqual(saved_accounts.count(), 1)
```

Comment:
わざわざ代入せずとも、以下のようにして実行することも可能です（以下同様）。
```suggestion
        self.assertEqual(Account.objects.all().count(), 1)
```